conversion/now.json

```diff
@@ -1,25 +1,14 @@
 {
   "version": 2,
-  "regions": [
-    "bru1"
-  ],
-  "builds": [
-    {
-      "src": "public/*",
-      "use": "@now/static"
+  "build": {
+    "env": {
+      "BA_HOST": "@ba-host"
     }
-  ],
-  "routes": [
-    {
-      "src": "/v1/bite.js",
-      "dest": "/public/bite.v1.js",
-      "headers": {
-        "cache-control": "max-age=31536000,immutable"
-      }
-    },
+  },
+  "rewrites": [
     {
-      "src": "/.*",
```

Comment:
I'll reimplement the catch-all route at a later point when the infrastructure is a bit more stable